sdcm/utils/common.py

```diff
@@ -2120,7 +2120,9 @@ def gce_download_dir(bucket, path, target):
     container = driver.get_container(container_name=bucket)
     dir_listing = driver.list_container_objects(container, ex_prefix=path)
     # first element is gce dir itself. Skip it.
```

Comment:
this comment started to lie

Reply:
Sound like we should have integration test for this function

Reply:
remove comment